src/ReverseProxy/Proxy/StreamCopyHttpContent.cs

```diff
@@ -68,7 +68,10 @@ public StreamCopyHttpContent(Stream source, bool autoFlushHttpClientOutgoingStre
         /// <see cref="HttpClient.SendAsync(HttpRequestMessage, HttpCompletionOption, CancellationToken)"/>
         /// completes, even when using <see cref="HttpCompletionOption.ResponseHeadersRead"/>.
         /// </remarks>
-        public bool Started { get; private set; }
+        public bool Started => Volatile.Read(ref _started) == 1;
```

Comment:
Nit: one test is using `== 1`, the other `!= 0`
```suggestion
        public bool Started => Volatile.Read(ref _started) != 0;
```

Reply:
I wanted to be consistent with here:
https://github.com/microsoft/reverse-proxy/blob/f6a268782a3dc8678536c95fe9ddfbacb8f076a3/src/ReverseProxy/Health/EntityActionScheduler.cs#L52
https://github.com/microsoft/reverse-proxy/blob/f6a268782a3dc8678536c95fe9ddfbacb8f076a3/src/ReverseProxy/Health/EntityActionScheduler.cs#L74

Should I change both?

Reply:
Oh, I see where you copied EntityActionScheduler. Yes, that one can also use Exchange and == 1.